secadvisor/pkg/gremlin_queries.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
```

Comment:
should be `pkg`

Reply:
You're right - this is wrong in the existing files on `master` (commit 90c7df9fa12ab110926d93ab2d5908303b100e0c renamed the files but didn't modify the `package mod` lines).

I'll submit a separate PR fixing all these lines to `package pkg`, and then later rebase this PR on top.

Reply:
https://github.com/skydive-project/skydive-flow-exporter/pull/14 (you can rebased on top of this)